maintainers.txt

```diff
@@ -3,4 +3,4 @@ h.drury@elifesciences.org
 c.donkin@elifesciences.org
 e.odier@elifesciences.org
 w.byrne@elifesciences.org
-#xpub-tech
```

Comment:
also here, shouldn't remove the Slack channel

Reply:
I thought this was a rogue comment.